app/assets/stylesheets/components/tags.scss

```diff
@@ -18,7 +18,7 @@
     opacity: 0.5;
   }
 
-  &:hover {
+  &[href]:hover {
```

Comment:
Is the addition of `[href]` because not all `.crayons-tag`s are anchor tags?

Reply:
Correct.